custom/icds_reports/tests/agg_tests/test_export_data.py

```diff
@@ -1106,330 +1106,6 @@ def test_awc_infrastructure_export(self):
-                            'st6',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered'
-                        ],
-                        [
-                            'st7',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered'
-                        ],
-                        [
-                            'st7',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered'
-                        ],
-                        [
-                            'st7',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered'
-                        ],
-                        [
-                            'st7',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered'
-                        ],
-                        [
-                            'st7',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered',
-                            'Data Not Entered'
-                        ]
-                    ]
-                ],
-                [
-                    "Export Info",
-                    [
-                        [
-                            "Generated at",
-                            "16:21:11 15 November 2017"
-                        ],
-                        [
-                            'State',
-                            'st1'
-                        ],
-                        [
-                            'District',
-                            'd1'
-                        ],
-                        [
-                            "Block",
-                            "b1"
-                        ]
-                    ]
-                ]
-            ]
-        )
-
-    def test_awc_infrastructure_export_with_beta(self):
-        self.assertListEqual(
-            AWCInfrastructureExport(
-                config={
-                    'domain': 'icds-cas'
-                }, beta=True
             ).get_excel_data(location=''),
```

Comment:
why the location is ='' not 'b1'?

Reply:
Removed it to check the default case. But better to folow the old test. SO added it back